imports/server/mediasoup.ts

```diff
@@ -335,12 +339,32 @@ class SFU {
 
     const observerAppData = observer.appData as AudioLevelObserverAppData;
 
-    const updateLastActivity = throttle(Meteor.bindEnvironment(() => {
-      CallHistories.upsert({
-        hunt: observerAppData.hunt,
-        call: observerAppData.call,
-      }, { $set: { lastActivity: new Date() } });
-    }), RECENT_ACTIVITY_TIME_WINDOW_MS);
+    const updateLastActivity = throttle(Meteor.bindEnvironment(
```

Comment:
Hmmm, if I'm reading this right, this broad throttling mechanism means we'll likely only ever see one user at a time with appreciable `volume`, since most of the time only one person is talking at a time.  That kinda misses the point of counting active users by number of unique speakers in a time interval.

We probably want, instead, to throttle writes to `RecentActivities` on a per-user basis.

Reply:
Yeah, you're right. Actually, I think we don't want to throttle the `RecentActivities` write at all, since we're already creating the audio level observer with `threshold: 100`.

Reply:
I assume you meant `interval: 100`.  If so:

Do we really want to be doing 10 writes/sec/speaker for each (speaking) audio call participant?  That seems like an order of magnitude more than anything else we've done to date.  I'd favor throttling activity tracking writes to, say, one write per second per user.  Having a short interval means we notice shorter bursts of conversation; having a long throttle period means we avoid excessive quantities of writes that contribute little to our understanding of activity.

I think it should be correct to do throttling in memory on the backend that provides the call's router -- for any given call, we're guaranteed to only observe call activity on that one backend.

Reply:
Yeah, OK, you're right - pushed a new version that throttles per-second-per-user.